src/d3a/models/market/one_sided.py

```diff
@@ -291,7 +304,8 @@ def accept_offer(self, offer_or_id: Union[str, Offer], buyer: str, *, energy: in
 
         if already_tracked is False:
             self._update_stats_after_trade(trade, offer)
-            log.info(f"[TRADE] [{self.name}] [{self.time_slot_str}] {trade}")
+            log.info(f"{self._debug_log_market_type_identifier}[TRADE] "
```

Comment:
Do you confirm that the level of this is actually `info`? I'm asking because of the `_debug_log_market_type_identifier` variable.

Reply:
It is info and it is important to stay that way. Otherwise you will not be able to see the trades in the default d3a log level